lib/active_model/serializable_resource.rb

```diff
@@ -64,5 +80,14 @@ def serializer?
     protected
 
     attr_reader :resource, :adapter_opts, :serializer_opts
+
+    def notify_active_support(&block)
+      return unless serializer?
+      event_name = 'render.active_model_serializers'
+      payload = { serializer: serializer, adapter: adapter }
+      ActiveSupport::Notifications.instrument(event_name, payload) do
+        block.call
+      end
```

Comment:
did you need to use the `&block` form?  I think you should be able to remove the method argument and replace block.call with `yield` (which is faster and saves object creation

we should also freeze the string here or reduce object allocations on what would be frequently called method, even if the notifications are turned off.

otherwise, :100: